docs/patterns.rst

```diff
@@ -1,12 +1,29 @@
 Patterns
 ========
 
+When you create a new Bottery project, it will contain a file called `patterns.py`.
+This file is were you will define what kind of messages your bot can receive and
+what kind of responses you should return. Just remeber that Bottery will check the patterns list 
+in order, so any generic message should be listed last, while more specific messages should have
+the patterns listed first.
+
+On your `pattern.py` file you should have a list named **patterns** containing different
+kind of pattern classes. By default, Bottery will import the `Pattern` class, which is the most
+simple type of pattern you can have: it should have a message that will "trigger" this pattern
+and a function that will tell Bottery what it should return. However, our plan is that 
+Bottery should allow you to have different kind of pattern classes that will allow you to have 
+regex, natural language processing tools, among others. 
+
 Pattern
 ^^^^^^^
 
-Pattern is a class that will receive a pattern message and a view. If the 
+*Pattern(pattern, view)* 
```

Comment:
Don't need this line here, we should link to the api reference (that we don't yet :P)

Reply:
Since we don't have the api reference for now, shouldn't it be better to have this here and remove it later?

Reply:
ok